apis/Google.Cloud.Firestore/Google.Cloud.Firestore/AggregateQuerySnapshot.cs

```diff
@@ -80,6 +80,9 @@ public T GetValue<T>(AggregateField aggregateField)
         return GetValue<T>(aggregateField.Alias);
     }
 
+    // Note: TryGetValue is internal right now, as we're not 100% convinced we want it in the public API.
```

Comment:
Hmmm, I'd leave it in, after all users can always catch the exception from GetValue and wrap it in their own TryGetValue.
I think this is different from when were talking about returning null given a non-existent alias.

Reply:
Discussed offline, and we'll keep it internal for now.